core/browser.py

```
@@ -174,6 +174,9 @@ async def ensure_browser(self) -> None:
             launch_kwargs: dict = {
                 "headless": True,
                 "args": list(_LAUNCH_ARGS),
+                # Playwright 的 Chromium 默认参数包含 --no-sandbox；仅从
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)